### PR TITLE
Jkyro/multi month calendar

### PR DESCRIPTION
**Pull request by jkyro**
Initial support for showing multiple months in calendar via 'months' option. If this is combined with dropdowns they are shown only on the first month. Includes some examples and tests.

### CODE REVIEW

File: docs/datepicker.md [hunk 1]
```diff
@@ -195,3 +195,6 @@ type: `string`
 
 type: `string`
 
+### `months`
```
Comment (martijnrusschen) [inline docs/datepicker.md]: You can ignore this, this document will be auto generated on every release.

File: src/calendar.jsx [hunk 2]
```diff
@@ -56,7 +57,8 @@ var Calendar = React.createClass({
 
   getDefaultProps () {
     return {
-      utcOffset: moment.utc().utcOffset()
+      utcOffset: moment.utc().utcOffset(),
+      months: 1
```
Comment (martijnrusschen) [inline src/calendar.jsx]: What about something more descriptive, like: `monthsShown`. Not the best naming but open for suggestions.

File: src/datepicker.jsx [hunk 3]
```diff
@@ -83,7 +84,8 @@ var DatePicker = React.createClass({
           attachment: 'together'
         }
       ],
-      utcOffset: moment.utc().utcOffset()
+      utcOffset: moment.utc().utcOffset(),
+      months: 1
```
Comment (martijnrusschen) [inline src/datepicker.jsx]: Same here.

File: src/datepicker.jsx [hunk 4]
```diff
@@ -218,6 +220,7 @@ var DatePicker = React.createClass({
         utcOffset={this.props.utcOffset}
         outsideClickIgnoreClass={outsideClickIgnoreClass}
         fixedHeight={this.props.fixedHeight}
+        months={this.props.months}
```
Comment (martijnrusschen) [inline src/datepicker.jsx]: And here.

**Comment by jkyro** [comment]
Thanks for the comments! I think the build fails because the added code lowers the coverage percent somewhat. Additional test cases should do the trick. As for the parameter naming, monthsShown sounds good to me, I'll fix that.

**Comment by martijnrusschen** [comment]
Thanks for adding this!

**Pull request merged**